index.html

```diff
@@ -38,7 +38,7 @@
 
         </div>
         <div id="label">
-            
+            <img src="./img/Google.png" alt="Google" class="img-google">
```

Comment:
This is an image with white text, putting it on a white background is not a good idea
I think it is a useless code

Reply:
ok. i will fix it. because my google have background color is black :))